web/components/contract/contract-prob-graph.tsx

```diff
@@ -84,13 +78,14 @@ export const ContractProbGraph = memo(function ContractProbGraph(props: {
           format: (time) => formatTime(+time, lessThanAWeek),
         }}
         colors={{ datum: 'color' }}
-        pointSize={bets.length > 100 ? 0 : 10}
+        curve="stepAfter"
+        pointSize={0}
         pointBorderWidth={1}
         pointBorderColor="#fff"
         enableSlices="x"
         enableGridX={!!width && width >= 800}
         enableArea
-        margin={{ top: 20, right: 28, bottom: 22, left: 40 }}
+        margin={{ top: 20, right: 0, bottom: 25, left: 40 }}
```

Comment:
The reason for the right margin is the labels get cut off:
<img width="742" alt="Screen Shot 2022-06-16 at 11 03 04 AM" src="https://user-images.githubusercontent.com/1593821/174115296-f2e33693-4a4e-40e6-95a9-59aed442185c.png">

Reply:
Ah interesting. Well with the other changes in my last commit, we don't need to get rid of the right margin anymore, I can put it back.